website/versioned_docs/version-v2.0_alpha/developers-references/maci-wrapper/QuickStartGuide.md

```diff
@@ -0,0 +1,243 @@
+---
+title: MACI Wrapper Guide Scaffold ETH 2 + MACI Voting Template
+description: Introduction to the MACI Wrapper and Setup
+sidebar_label: Quickstart Guide
+sidebar_position: 1
+---
+
+# MACI Wrapper Guide: Scaffold ETH 2 + MACI Voting Template
+
+Welcome to the hacker guide for the Scaffold ETH 2 + MACI Voting Template! This guide will help you get started with the template, providing step-by-step instructions to set up and customize your decentralized voting application, understand MACI, and explore project ideas.
```

Comment:
let's add here a link to the repo and add your name + link as author (if you wish to) 
something like
[Scaffold ETH 2 + MACI Voting Template](https://github.com/yashgo0018/maci-wrapper) by [Yash](https://github.com/yashgo0018)  ?

Reply:
Added the link and the author info